lcmtypes/lcmt_trajectory_block.lcm

```diff
@@ -0,0 +1,12 @@
+package dairlib;
+
+struct lcmt_trajectory_block
+{
+  string trajectory_name;
+  
+  int32_t num_points;
+  int32_t num_datatypes;
+  double time_vec[num_points];
+  double datapoints[num_points][num_datatypes];
```

Comment:
It will be helpful to add comments on what `dataypes` is here, since I'm not very sure what the second dimension of datapoints is here. Maybe the name `datatypes` confuses me. 
I guess `datapoints` stores a `num_datatypes`-size vector v(t)? For example, an input u(t) with n_u number of actuation.

Reply:
You shouldn't have to interface with this lcm object, I'm using it as a convenience to store/load using the LCM serializer. Although that's not the case for lcmt_metadata object.